src/wp-includes/rest-api/endpoints/class-wp-rest-themes-controller.php

```diff
@@ -318,7 +318,7 @@ public function prepare_item_for_response( $item, $request ) {
 
 		if ( $theme->get_stylesheet() === wp_get_theme()->get_stylesheet() ) {
 			// This creates a record for the current theme if not existent.
-			$id = WP_Theme_JSON_Resolver::get_user_custom_post_type_id();
+			$id = WP_Theme_JSON_Resolver::get_global_styles_post_id();
 		} else {
 			$user_cpt = WP_Theme_JSON_Resolver::get_user_data_from_custom_post_type( $theme );
```

Comment:
```suggestion
			$user_cpt = WP_Theme_JSON_Resolver::get_global_styles_from_post( $theme );
```
Missed this one when resolving the merge conflicts.